templates/email/components/unsubscribe.ejs

```diff
@@ -0,0 +1,12 @@
+<tr>
+    <td align="center" bgcolor="#220050" style="padding:0;Margin:0;padding-top:25px">
+        <p style="Margin:0;-webkit-text-size-adjust:none;-ms-text-size-adjust:none;mso-line-height-rule:exactly;font-family:'open sans', 'helvetica neue', helvetica, arial, sans-serif;line-height:19.2px;color:#daf0ff;font-size:16px">Best regards,<br>Kleros Zodiac Notifications Bot.</p>
+    </td>
+</tr>
+<tr>
+    <td align="center" bgcolor="#220050" style="padding:0;Margin:0;padding-top:10px;padding-bottom:25px">
+        <a href="mailto:<%- unsubscribeEmail %>?subject=Unsubscribe%20Request&body=Hello,%0D%0A%0D%0AI%20would%20like%20to%20request%20to%20unsubscribe%20from%20receiving%20further%20emails.%0D%0A%0D%0AThank%20you." target="_blank">
+            <p style="Margin:0;-webkit-text-size-adjust:none;-ms-text-size-adjust:none;mso-line-height-rule:exactly;font-family:'open sans', 'helvetica neue', helvetica, arial, sans-serif;line-height:19.2px;color:#6cc5ff;font-size:16px"><u>Unsubscribe</u></p>
```

Comment:
Is this the button?

Reply:
@wadader yes, that is the button. Honestly, the HTML is not very pleasant to read, but it is required for email client compatibility